### DISCUSSION TITLE
Pivot Point Not Accessible

### DISCUSSION THREAD

**Discussion by CircuitJunkie** [keiwando/toonsquid#569] [General]
It may be that I’m doing something wrong, but when I enable pivot editing mode, I do not see the pivot marker. It is possible that the pivot point is buried under the transform center for the object. I am currently attempting to change the pivot point on an ellipse.

**Comment by Cat-Butler** [comment]
Are you talking about a temporary pivot for editing (e.g. "that eye's on a funny angle") or pivot point for animation?
The first relies on Actions > Settings > Tools > Temporary pivots enabled. 
My personal preference is to turn these off, so I don't get confused with animation pivots.

The second relies on (a) being in keyframe mode (timeline showing, keyframe icon highlighted) and also (b) not being in timeline edit/drawing mode (the pen or pixel icon near the top right of the screen **not** being highlighted). The second one is the one that catches a lot of people out (myself included).

And, as always, I could be completely off-track. Best wishes!
